test/dynamo/test_functions.py

```diff
@@ -819,6 +823,24 @@ def augment(x: torch.Tensor) -> torch.Tensor:
     #         case {"b": param}:
     #             return x / param
 
+    @make_test
+    @requires_numpy_pytorch_interop
+    def test_numpy_meshgrid(x, y):
```

Comment:
wait, how did we generate these input tensors 🤔

Reply:
Maybe [here](https://github.com/pytorch/pytorch/blob/master/torch/_dynamo/testing.py#L213)